### ISSUE TITLE
[BUG] Dict[] type annotations make pytkdocs explode

### ISSUE THREAD

**Issue by nigelm** [pawamoy/pytkdocs#73]
**Describe the bug**
If there is a type annotation of the form `Dict[x, y]` then the parser throws an exception.

**To Reproduce**
```python
from typing import Any
from typing import Dict
...
my_thing: Dict[str, Any] = {}
```

**Expected behavior**
For it not to throw an exception like this:

```
INFO    -  Building documentation...
INFO    -  Cleaning site directory
ERROR   -  mkdocstrings.handlers.python: Collection failed: 'Tuple' object has no attribute 'value'
Traceback (most recent call last):
  File "/Users/nigel/Library/Caches/pypoetry/virtualenvs/python-broadworks-tools-zQ4enKcE-py3.9/src/pytkdocs/src/pytkdocs/cli.py", line 205, in main
    output = json.dumps(process_json(line))
  File "/Users/nigel/Library/Caches/pypoetry/virtualenvs/python-broadworks-tools-zQ4enKcE-py3.9/src/pytkdocs/src/pytkdocs/cli.py", line 114, in process_json
    return process_config(json.loads(json_input))
  File "/Users/nigel/Library/Caches/pypoetry/virtualenvs/python-broadworks-tools-zQ4enKcE-py3.9/src/pytkdocs/src/pytkdocs/cli.py", line 91, in process_config
    obj = loader.get_object_documentation(path, members)
  File "/Users/nigel/Library/Caches/pypoetry/virtualenvs/python-broadworks-tools-zQ4enKcE-py3.9/src/pytkdocs/src/pytkdocs/loader.py", line 342, in get_object_documentation
    root_object = self.get_module_documentation(leaf, members)
  File "/Users/nigel/Library/Caches/pypoetry/virtualenvs/python-broadworks-tools-zQ4enKcE-py3.9/src/pytkdocs/src/pytkdocs/loader.py", line 409, in get_module_documentation
    root_object.add_child(self.get_class_documentation(child_node))
  File "/Users/nigel/Library/Caches/pypoetry/virtualenvs/python-broadworks-tools-zQ4enKcE-py3.9/src/pytkdocs/src/pytkdocs/loader.py", line 444, in get_class_documentation
    attributes_data.update(get_instance_attributes(class_.__init__))
  File "/Users/nigel/Library/Caches/pypoetry/virtualenvs/python-broadworks-tools-zQ4enKcE-py3.9/src/pytkdocs/src/pytkdocs/parsers/attributes.py", line 155, in get_instance_attributes
    annotation = node_to_annotation(assignment)
  File "/Users/nigel/Library/Caches/pypoetry/virtualenvs/python-broadworks-tools-zQ4enKcE-py3.9/src/pytkdocs/src/pytkdocs/parsers/attributes.py", line 19, in node_to_annotation
    return f"{node.annotation.value.id}[{node_to_annotation(node.annotation.slice.value)}]"  # type: ignore
AttributeError: 'Tuple' object has no attribute 'value'

ERROR   -  mkdocstrings.extension: Could not collect 'python_broadworks_tools.enterprise'
ERROR   -  Error reading page 'api/enterprise.md': 'Tuple' object has no attribute 'value'
```

**System (please complete the following information):**
- `pytkdocs` version 0.9.0 and github version
- Python version: 3.8.5
- OS: Mac

**Comment by pawamoy** [comment]
Hello, thank you for the report!

I'll try to replicate first, then look for a fix (PRs welcome!) :slightly_smiling_face:

**Comment by phil65** [comment]
This only seems to happen on Python 3.9.x. (The bug report mentions the wrong version)

**Comment by pawamoy** [comment]
Thanks @phil65 for the note. It must then be related to #75: `ast` has changed in Python 3.9, and it breaks the current pytkdocs parser. These bugs are preventing the use of Python 3.9 so I'll mark this (in my head) as pretty urgent.

**Comment by pawamoy** [comment]
I've pushed a fix, could you try again with the latest commit?

**Comment by pawamoy** [comment]
Should be fixed in 0.10.1.
